frontend/src/components/SamplesList.tsx

```diff
@@ -109,8 +109,7 @@ export default function SamplesList({
     refetchSearchVal().then(() => {
       gridRef.current?.api?.hideOverlay();
     });
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [parsedSearchVals]);
+  }, [parsedSearchVals, columnDefs, refetchWhereVariables, refetch]);
```

Comment:
Updated the variable dependencies that trigger the refetching of data. The most important one here being `columnDefs` which defines the columns shown to users. So whenever users click on the "View WES samples" tab, the dashboard will refetch (currently, it does not do that).